packages/popmotion-pose/docs/api/react/posed.md

```diff
@@ -4,8 +4,6 @@ description: Create a posed component
 category: react
 ---
 
-**Note:** React Pose is built with React 16.3.0, which is currently in alpha.
```

Comment:
16.3.0 was released this morning 🎉

Reply:
I was about to create a PR for this right now :smile: 
But I think it should be better to only remove the part of the sentence behind the comma instead of removing the whole line. Otherwise some users might be confused when trying to use it with React < 16.3.

Reply:
@arcticicestudio This PR also adds the backwards compatibility polyfill so it should work with older versions of React